util/tmp_file.go

```diff
@@ -0,0 +1,18 @@
+package util
+
+import (
+	"os"
+)
+
+// Clean up a created temporary file, reporting any errors to the log
+func CleanupTmpFileAndLogErrors(tmpFile *os.File, logger *Logger) {
```

Comment:
```suggestion
func CleanupTmpFile(tmpFile *os.File, logger *Logger) {
```
NBD, but I think if we're passing a logger, it's pretty obvious we're going to be logging something with it, so I'd just drop it :man_shrugging: .

Reply:
Actually, to be totally pedantic, I suppose this should be:
```suggestion
func CleanUpTmpFile(tmpFile *os.File, logger *Logger) {
```
:laughing: